pagopa/io-app-design-system
# feat: [IOBP-186] Import updated `ListItemTransaction` component

## Short description
This PR imports the new updated version of `ListItemTransaction` component and adds a new prop `badgeText` in order to show a text inside when the badge is visible (useful for translated text)

## List of changes proposed in this pull request
- Added `ListItemTransaction` component and its utilities;
- Added a new propr `badgeText` which is required only when the status of a transaction is: `"failure" | "pending" | "cancelled" | "reversal"`

## How to test
Open the ListItem section and check if the ListItemTransaction is correctly showed;

## Preview
https://github.com/pagopa/io-app-design-system/assets/34343582/e8a786ce-a229-482d-811e-3b7d7160076c
